ui_components/src/chat/Chat_new.js

```diff
@@ -0,0 +1,385 @@
+                                    </a>
+                                    <div className="post_village_title">
+                                        {post.userTitle.toLowerCase()}
+                                        <img
+                                            className='chat_village_icon'
+                                            src={`./images/village_icons/${post.userVillage.toLowerCase()}.png`}
+                                            alt={`${post.userVillage} Village`}
+                                            title={`${post.userVillage} Village`}
+                                        />
+                                    </div>
+                                    {post.staffBannerName &&
+                                        <div className="post_staff_title" style={{ backgroundColor: post.staffBannerColor }}>
+                                            {post.staffBannerName}
+                                        </div>
+                                    }
+                                </div>
+                            </div>
+                        </div>
+                        <div className="post_message_container ft-s ft-c3 ft-default" data-id={post.id} dangerouslySetInnerHTML={{ __html: post.message }}>
+                        </div>
+                        <div className="post_controls_container ft-s ft-c3">
+                            <div className="post_controls_wrapper">
+                                <div className='post_quote_wrapper'><img
+                                    className='post_quote'
+                                    src='../images/v2/icons/quote_hover.png'
+                                    onClick={() => quotePost(post.id)}
+                                /></div>
+                                {isModerator &&
+                                    <div className='post_delete_wrapper'>
+                                        <img
+                                            className='post_delete'
+                                            src='../images/v2/icons/delete_hover.png'
+                                            onClick={() => deletePost(post.id)}
+                                        />
+                                    </div>
+                                    
+                                }
+                                <div className='post_report_wrapper'>
+                                    <a className='imageLink' href={post.reportLink}>
+                                        <img className='post_report' src='../images/v2/icons/report_hover.png' />
+                                    </a>
+                                </div>
+                            </div>
+                            <div className="post_timestamp">
+                                <div style={{ marginBottom: "2px" }}>{post.timeString}</div>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
This should all get split out into a `ChatPost` component

Reply:
Marking as to-do.